### PR TITLE
Initial `PlayerControls.podspec`

### PR DESCRIPTION
**Pull request by AndriiMoskvin**

### CODE REVIEW

File: PlayerControls.podspec [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                       DESC
+
+  s.homepage         = "https://github.com/vidible/OneMobileSDK-controls-ios"
+  s.author           = {
+    "Andrey Moskvin" => "andrey.moskvin@teamaol.com",
+    "Alexey Demedetskiy" => "alexey.demedetskiy@teamaol.com"
+  }
+
+  s.source           =  { :git => 'https://github.com/vidible/OneMobileSDK-controls-ios.git', :tag => '1.0' }
+  s.source_files     = "PlayerControls/sources/*"
```
Comment (AlexeyDemedetskiy) [inline PlayerControls.podspec]: isnt xib need to be separated?

Comment (AlexeyDemedetskiy) [inline PlayerControls.podspec]: Where is *.xcasset mention?

Comment (AndriiMoskvin) [inline PlayerControls.podspec]: I've forgot about resources, I will add them.

Comment (AndriiMoskvin) [inline PlayerControls.podspec]: I've forgot about resources, I will add them.

**Comment by AndriiMoskvin** [comment]
@AlexeyDemedetskiy Fixed and checked.

**Pull request merged**